apps/calendar/js/app.js

```diff
@@ -304,31 +327,24 @@ module.exports = {
    */
   init: function() {
     debug('Will initialize calendar app...');
-    var self = this;
-    var pending = 2;
-
-    function next() {
-      pending--;
-      if (!pending) {
-        self._init();
-      }
-    }
 
     if (!this.db) {
       this.configure(new Db('b2g-calendar', this), new Router(page));
     }
 
-    // start the workers
-    this.serviceController.start(false);
-
-    navigator.mozL10n.once(next);
     this.db.load(() => {
-      next();
+      this._initControllers();
       // it should only start listening for month change after we have the
       // calendars data, otherwise we might display events from calendars that
       // are not visible. this also makes sure we load the calendars as soon as
       // possible
-      this.store('Calendar').all(() => dayObserver.init());
+      this.store('Calendar').all(() => {
```

Comment:
see the old mark calls: https://github.com/millermedeiros/gaia/commit/b55d0ee024e51fce9e25188b2529fce2c9c40718